inc/audit_log_to_cloudwatch/class-cloudwatch-driver.php

```diff
@@ -0,0 +1,216 @@
+<?php
```

Comment:
missing file header comment

Reply:
We don't really have these elsewhere. Is this a requirement? Examples in our coding guidelines maybe?

Reply:
Oh, genuinely thought it was part of our coding standards. I've been doing it out of habit, a brief description and a `@package hm-platform/<module>` line.